literature.bib

```diff
@@ -6797,6 +6992,38 @@ @inproceedings{RAK+:VaMoS13
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2012 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{LCF+:TASE12,
+	title={A Succinct and Efficient Implementation of a 2\^{} 32 BDD Package},
+	author={Lv, Guanfeng and Chen, Yao and Feng, Yachao and Chen, Qingliang and Su, Kaile},
+	booktitle={Proc.\ Int'l Symposium on Theoretical Aspects of Software Engineering (TASE)},
+	pages={241--244},
+	publisher=IEEE,
+	address = Washington,
+	year=2012,
+}
+
+@article{M:JSW12,
+	title={Biddy - a Multi-platform Academic {BDD} Package},
+	author={Meolic, Robert},
+	journal={J.\ of Software (JSW)},
+	volume={7},
+	number={6},
+	pages={1358--1366},
+	publisher = {{International Academy Publishing (IAP)}},
+	address = {San Bernardino, CA, USA},
+	year = 2012
```

Comment:
Why no shorthand for journal, publisher, address?

Reply:
First occurrence of journal, publisher, address.